### ISSUE TITLE
disable adblocker

### ISSUE THREAD

**Issue by ghajini** [uBlockOrigin/uAssets#117]
Replace the bracketed [...] placeholders with your own information.

### URL(s) where the issue occurs

http://m.mid-day.com/articles/bundle-of-joy-arrives-its-a-girl-for-shahid-kapoor-mira-rajput/17563182
### Describe the issue

- go to m.mid-day.com on firefox android with ublock origin

- click any news

- on going to site following message displayed
" Blocked
Hi, it seems there is some blocker enabled that is hampering the content to display.kindly disable or remove it to view the content "

### Screenshot(s)

![screenshot_2016-08-26-23-09-38](https://cloud.githubusercontent.com/assets/20338483/18014807/483f0cae-6be2-11e6-8b79-ab64d296ba78.png)

### Versions

- Browser/version: firefox 48 for android
- uBlock Origin version: 1.8.4

### Settings

-  uBO's default settings+adblock warning removal list+antiadblock killer

### Notes

i clicked a feature in browser ''request desktop site'' to tell site to use user agent same as pc(so that it is same as ublock on pc) but site redirects to mobile version snd detects adblocker

- site display antiadblock notice for m.mid-day.com AND NOT for www.mid-day.com

basically issue happens when accesing mobile version of website(iam using android smartphone with ublock orign on firefox)

@gorhill 
please open another repository for reporting issues on ublock origin for firefox for android

**Comment by IDKwhattoputhere** [comment]
Try using [the logger](https://github.com/gorhill/uBlock/wiki/The-logger) next time. I found this filter in 10 seconds that way. Try it.
`||mid-day.com/*/adblockhandlega.js`
